runner/executer.go

```diff
@@ -0,0 +1,67 @@
+package runner
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"time"
+
+	"github.com/cybozu-go/github-actions-controller/metrics"
+)
+
+type Executer interface {
+	RunConfigure(ctx context.Context, configArgs []string) error
+	RunService(ctx context.Context) error
+}
+
+type ExecuterImpl struct {
```

Comment:
The name "ExecuterImpl" is very abstract.
We can't know what ExecuterImpl executes.
You should give it a concreate name.

Reply:
I don't know if it's an appropriate name, but I changed it to `listener`.